spec/checkout_sdk/disputes/disputes_previous_integration_spec.rb

```diff
@@ -0,0 +1,266 @@
+RSpec.describe CheckoutSdk::Disputes do
```

Comment:
Instead of calling this as `previous` how about placing this inside a folder `previous` with the same name.
See #64  for reference
wdyt?